proposals/low-level-struct-improvements.md

```diff
@@ -1491,9 +1503,9 @@ S Usage()
 }
 ```
 
-To make these APIs usable the compiler has to create separation between how freely `ref` and their values can be returned by default. This is the rational for having `ref` to `ref struct` and `out` be [implicitly scoped](#implicitly-scoped).
+To make these APIs usable the compiler ensures that `ref` have a smaller lifetime than their associated parameters. This is the rational This is the rational for having *ref-safe-to-escape* for `ref` to `ref struct` be *return only* and `out` be *containing method*. That prevents cyclic assignment because of the difference in lifetimes.
```

Comment:
Looks like *this is the rational* is repeated and was probably meant to be *this is the rationale*.

Reply:
```suggestion
To make these APIs usable the compiler ensures that `ref` have a smaller lifetime than their associated parameters. This is the rationale for having *ref-safe-to-escape* for `ref` to `ref struct` be *return only* and `out` be *containing method*. That prevents cyclic assignment because of the difference in lifetimes.
```

Reply:
Are you accusing me of mis-spelling a word? ... yeah I'm guilty